perfkitbenchmarker/centurylink/clc_virtual_machine.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ./pkb.py --cloud=Centurylink --machine_type=UBUNTU-12-64-TEMPLATE
+# --benchmarks=iperf
+
+from perfkitbenchmarker import linux_virtual_machine
+from perfkitbenchmarker import virtual_machine
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.centurylink import clc_disk
+from perfkitbenchmarker.centurylink import util
+
+
+class CenturylinkVirtualMachine(virtual_machine.BaseVirtualMachine):
+    """Object representing a Centurylink Virtual Machine."""
+
+    DEFAULT_ZONE = 'NY1'
+    DEFAULT_MACHINE_TYPE = 'UBUNTU-12-64-TEMPLATE'
+    DEFAULT_IMAGE = 'ubuntu-14-04-x64'
+
+    def __init__(self, vm_spec):
+        """Initialize a Centurylink virtual machine."""
+
+        super(CenturylinkVirtualMachine, self).__init__(vm_spec)
+        self.server_name = ''
+        self.disk_size = 1
```

Comment:
In that  case can you please use the disk spec's disk_size attribute instead of this? (CreateScratchDisk() is passed a disk spec when a new scratch disk is created)